antora-ui-camel/src/css/frontpage.css

```diff
@@ -217,6 +218,11 @@ section.frontpage h1 {
   width: 27rem;
 }
 
+.evaIcons {
+  width: 60px !important;
+  filter: invert(24%) sepia(38%) saturate(1353%) hue-rotate(200deg) brightness(89%) contrast(99%);
```

Comment:
Couldn't we change the color in the icon, for other svg icons we style them with CSS using a technique described [here](https://css-tricks.com/cascading-svg-fill-color/). Have a look at this bit in CSS:

https://github.com/apache/camel-website/blob/3120c065b92dfca83059551d416a859134b89462/antora-ui-camel/src/css/header.css#L391-L396

And [this SVG file](https://github.com/apache/camel-website/blob/master/antora-ui-camel/src/img/brand-logos.svg?short_path=ec92056).